api/v1alpha1/clusterextension_types.go

```diff
@@ -113,8 +114,9 @@ type PreflightConfig struct {
 // CRDUpgradeSafetyPreflightConfig is the configuration for CRD upgrade safety preflight check.
 type CRDUpgradeSafetyPreflightConfig struct {
 	//+kubebuilder:Required
-	// Disabled represents the state of the CRD upgrade safety preflight check being disabled/enabled.
-	Disabled bool `json:"disabled,omitempty"`
+	// +kubebuilder:validation:Enum:="Enabled,Disabled"
```

Comment:
This would make the only allowed value the exact string `Enabled,Disabled`. I think we are looking for:
```suggestion
	// +kubebuilder:validation:Enum:="Enabled","Disabled"
```

Reply:
Good catch. I've updated this in the new commit. Thanks